hindi/historical-places/taj-mahal.md

```diff
@@ -0,0 +1,3 @@
+ताजमहल (उर्दू: تاج محل) भारत के आगरा शहर में स्थित एक विश्व धरोहर मक़बरा है। इसका निर्माण मुग़ल सम्राट शाहजहाँ ने, अपनी पत्नी मुमताज़ महल की याद में करवाया था।[1] ताजमहल मुग़ल वास्तुकला का उत्कृष्ट नमूना है। इसकी वास्तु शैली फ़ारसी, तुर्क, भारतीय और इस्लामी वास्तुकला के घटकों का अनोखा सम्मिलन है।
```

Comment:
This line is longer than 120 characters. Could you break it up into multiple lines? That way it will be easier to read the raw files or view diffs on the command line.  No need to add blank lines in between, just press enter somewhere in the middle of the line.